Detectors/TRD/base/include/TRDBase/DigitIndex.h

```diff
@@ -0,0 +1,48 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ALICEO2_TRD_DIGITINDEX_H_
+#define ALICEO2_TRD_DIGITINDEX_H_
+
+namespace o2
+{
+namespace trd
+{
+
+class DigitIndex
+{
+ public:
+  DigitIndex() = default;
```

Comment:
I am not sure how C++ initialises the padded bits. I would assume there is no guarantee that they will be set to zero, and in this case ROOT I/O might not be able to compress them out. 
In this case, we should set them to zero explicitly in the constructor (which would require a new bitfield that occupies all bits that would be padded).

Reply:
I agree with you - but maybe this isn't really required since at Digitizer level all digit indexes are being fully constructed (with their elements). 

Anyway we should keep an eye because it will be possible since C++20 and that implementation would look nice because it is preferred to have member initialization rather than using the constructor for member initialization.

Reply:
It might be sufficient and/or necessary to add a bitfield variable that covers the unused bits to a total size of 32 bits. The default constructor would probably set this to zero, but the behaviour for pure padding is probably undefined.

Reply:
I understand you point, but after today's meeting and the test I just provided (see Jira ticket), I think that we should just stick with a regular data structure using unsigned integers (one of size 16 and two of size 8), as you proposed also.